patterns/explicit-construction.md

```diff
@@ -0,0 +1,46 @@
+##The Explicit Construction Anti-Pattern
```

Comment:
Some hash or double? It is not consistent with the other file. Also, there should be a space after the hash for readability right?